spec/support/stripe_payment_method_helper.rb

```diff
@@ -42,7 +36,7 @@ def to_stripejs_wallet_payment_method
     end
 
     def to_stripejs_payment_method_id
-      to_stripejs_payment_method.id
+      self[:payment_method_id] || to_stripejs_payment_method.id
```

Comment:
If a `payment_method_id` is present use that directly, otherwise create a new payment method on Stripe with the test token. Earlier a new payment method was created using the raw card number.